### PR TITLE
Sort rat excludes alphabetically

### PR DESCRIPTION
**Pull request by junichi11**
To prevent to add duplicate one, rat excludes should be sorted alphabetically. Actually, there was one duplicate line.

### CODE REVIEW

File: nbbuild/build.xml [hunk 1]
```diff
@@ -1985,7 +1985,6 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
                 <exclude name="javafx2.project/src/org/netbeans/modules/javafx2/project/templates/**" /> <!--license would be visible when users edit the templates inside their IDE-->
                 <exclude name="utilities/test/unit/src/org/netbeans/modules/openfile/resources/recent_files/*" /> <!--test data-->
                 <exclude name="**/*.url" /> <!--no degree of creativity-->
-		<exclude name="spi.navigator/test/unit/src/org/netbeans/modules/navigator/resources/lastsel/file.lastsel_mime1" /> <!--empty file-->
```
Comment (junichi11) [inline nbbuild/build.xml]: This line starts with TAB. So, I removed this line.

**Comment by emilianbold** [comment]
I like the idea but this will cause conflicts on all the open PRs that add an exclude. So, I'd rather we do this at the end of the module review. Even if a duplicate sneaks in, it's harmless.

**Comment by junichi11** [comment]
> So, I'd rather we do this at the end of the module review.

Yes, We need not hurry to do this at all :)

**Comment by geertjanw** [comment]
I don't think there are any PR's right now that have excludes, or am I wrong?

If true, can we merge this one, have more to add to it, but would be good to get this one in.

**Comment by junichi11** [comment]
I've fixed. #163 and https://github.com/emilianbold/incubator-netbeans/pull/324 have excludes. But it seems that they already conflict. If there are no problems, I'll merge this later.

**Comment by geertjanw** [comment]
One reason I’d like this one to be merged is that I have unit test data to be added to the Rat excludes (from schema2beans and java.hints) and this pr cleans and organizes the Rat excludes very well.

**Comment by junichi11** [comment]
I'll merge soon.

**Pull request merged**